lib/hypervisor/hv_kvm/monitor.py

```diff
@@ -811,6 +771,21 @@ def GetCpuInformation(self):
 
     return self.Execute("query-cpus-fast")
 
+  @_ensure_connection
+  def GetHotpluggableCPUs(self):
```

Comment:
not sure why this method gets added here

Reply:
removed